movo_7dof_moveit_config/launch/ompl-chomp_planning_pipeline.launch.xml

```diff
@@ -0,0 +1,19 @@
+<launch>
```

Comment:
Where did these planners come from? Are they useful? they seem to be different planners but I thought we only use OMPL